pkgs/applications/misc/dupeguru/default.nix

```diff
@@ -0,0 +1,62 @@
+{stdenv, python, gettext, qt5, fetchFromGitHub}:
```

Comment:
Better just use python3.

Reply:
@jtojnar what about https://github.com/NixOS/nixpkgs/pull/75054#discussion_r355661631 ? Is it acceptable advice?

Reply:
The advice about using only one of `pythonXxx` or `pythonXxxPackages` is sound. Otherwise, when user overrides one of them and forgets to override the other one, you would get multiple Pythons.